schemas/iso19110/src/main/plugin/iso19110/formatter/xsl-view/view.xsl

```diff
@@ -305,6 +306,16 @@
     <xsl:value-of select="."/>
   </xsl:template>
 
+  <xsl:template mode="render-value" match="gmx:Anchor">
+    <xsl:choose>
+      <xsl:when test="starts-with(./@xlink:href,'http')">
+        <a href="{./@xlink:href}"><xsl:value-of select="gfc:code/*/text()"/></a>
+      </xsl:when> 
+      <xsl:otherwise><xsl:value-of select="./text()"/></xsl:otherwise>
```

Comment:
Here you match the text and 2 lines above you match `gfc:code/*/text()`, is this normal ? Also if expecting `gfc:code`, maybe the match of the template should be `gfc:code/gmx:Anchor` ?

Reply:
good catch, this is not going to work like this